<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankWindow</class>
 <widget class="QMainWindow" name="RankWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>结果展示</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>631</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="DealerPoint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>庄家点数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="DealerBJ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>庄家是否实现blackjack:</string>
     </property>
    </widget>
    <widget class="QLabel" name="DealerPointValue">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="DealerBJValue">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="PlayerPoint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>玩家1点数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerBJ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>玩家1是否实现blackjack:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerPointValue">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerBJValue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="PlayerPoint_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>玩家2点数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerBJ_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>玩家2是否实现blackjack:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerPointValue_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerBJValue_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="PlayerPoint_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>玩家3点数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerBJ_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>玩家3是否实现blackjack:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerPointValue_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlayerBJValue_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="value1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-10</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>庄家vs玩家1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>庄家vs玩家2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>庄家vs玩家3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="result1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="result2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="result3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
